site/app/controllers/grading/popup_refactor/RubricGraderController.php

```diff
@@ -87,18 +131,22 @@ public function createMainRubricGraderPage(
         string $direction = "ASC",
         string $navigate_assigned_students_only = "true"
     ) {
-
-        $this->setMemberVariables($gradeable_id, $who_id, $sort, $direction, $navigate_assigned_students_only);
+        $this->setMemberVariables($gradeable_id, $who_id, $sort, $direction);
```

Comment:
`$this->setMemberVariables` calls the `setCurrentSubmission` function which can possibly set `$this->current_submission` to false if it fails to get the gradeable for whatever reason. Even though it attempts to add a redirect to the page, the renderOutput function is still calling the `createRubricGradeableView` which is expecting a GradedGradeable for the $current_submission variable but could possibly be false